src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -769,24 +797,39 @@ void BLEManagerImpl::HandleConnect(struct bt_conn * conId, uint8_t err)
 {
     ChipDeviceEvent event;
 
+    PlatformMgr().LockChipStack();
+
+    // Don't handle BLE connecting events when it is not related to CHIPoBLE
+    VerifyOrExit(ConnectivityMgr().IsBLEAdvertisingEnabled(), );
```

Comment:
nit: the lock could be release just after this call. `PostEvent` doesn't need to be guarded.